extensions/objectify/pom.xml

```diff
@@ -0,0 +1,151 @@
+      <version>5.1.21</version>
+      <classifier>sources</classifier>
+    </dependency>
+
+    <!-- Google App Engine -->
+    <dependency>
+      <groupId>com.google.appengine</groupId>
+      <artifactId>appengine-api-1.0-sdk</artifactId>
+      <version>${gae.version}</version>
+      <scope>compile</scope>
+    </dependency>
+    <dependency>
+      <groupId>com.google.appengine</groupId>
+      <artifactId>appengine-api-labs</artifactId>
+      <version>${gae.version}</version>
+      <scope>provided</scope>
+    </dependency>
+
+    <dependency>
+      <groupId>com.fasterxml.jackson.core</groupId>
+      <artifactId>jackson-databind</artifactId>
+      <scope>provided</scope>
+    </dependency>
+
+    <!-- Test dependencies -->
+    <dependency>
+      <groupId>junit</groupId>
+      <artifactId>junit</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>com.google.appengine</groupId>
+      <artifactId>appengine-api-stubs</artifactId>
+      <version>${gae.version}</version>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>com.google.appengine</groupId>
+      <artifactId>appengine-testing</artifactId>
+      <version>${gae.version}</version>
+      <scope>test</scope>
+    </dependency>
+  </dependencies>
+
+  <build>
+    <plugins>
+      <plugin>
+        <artifactId>maven-surefire-plugin</artifactId>
+        <configuration>
+          <excludes>
+            <exclude>**/client/**/*TestCase.java</exclude>
+          </excludes>
+        </configuration>
+      </plugin>
+
+      <plugin>
+        <groupId>org.codehaus.mojo</groupId>
+        <artifactId>gwt-maven-plugin</artifactId>
+        <executions>
+          <execution>
+            <id>packaging</id>
+            <goals>
+              <goal>resources</goal>
+            </goals>
+            <configuration>
+              <modules>
+                <module>com.github.nmorel.gwtjackson.objectify.GwtJacksonObjectify</module>
+              </modules>
+            </configuration>
+          </execution>
+          <execution>
+            <id>test-dev-mode</id>
+            <phase>test</phase>
+            <goals>
+              <goal>test</goal>
+            </goals>
+            <configuration>
+              <module>com.github.nmorel.gwtjackson.objectify.GwtJacksonObjectifyTest</module>
+              <out>${project.build.directory}/gwt/test/dev</out>
+            </configuration>
+          </execution>
+          <execution>
+            <id>test-prod</id>
+            <phase>test</phase>
+            <goals>
+              <goal>test</goal>
+            </goals>
+            <configuration>
+              <module>com.github.nmorel.gwtjackson.objectify.GwtJacksonObjectifyTest</module>
+              <out>${project.build.directory}/gwt/test/prod</out>
+              <productionMode>true</productionMode>
+            </configuration>
+          </execution>
+        </executions>
+      </plugin>
+      <plugin>
```

Comment:
Do you need 1.7 ? 
Can you try by just setting the following properties on top the files (like in pom examples) : 
```xml
    <maven.compiler.source>1.7</maven.compiler.source>
    <maven.compiler.target>1.7</maven.compiler.target>
```

Reply:
Sure!
Coming in few minutes

Reply:
And no, it doesn't require 1.7, so I will just remote them

Reply:
@nmorel 
I want to keep the PR clean, there are right now 3 commits in this PR:
- make generic key works
- Create the Objectify extension
- Create the Remote Logging extension

Is it OK for you if I amend commits to keep only these 3 commits?

Reply:
Fixed!

Reply:
Yes sure. You can rebase or squash :)